source/classes.rst

```diff
@@ -4,6 +4,11 @@ Classes
 Python 3 drops support for “old-style” classes, and introduces dedicated syntax
 for metaclasses. Read on for details.
 
+
+.. index:: object; as base class
```

Comment:
I don't think that's necessary? I think it's assumed that such sub-entries in an index relate to the parent entry. See “raise; new syntax” for example.
But if you think it's clearer I can add it.